tools/test_configs/config_paths.json

```diff
@@ -13,5 +13,6 @@
     "6LOWPAN_ROUTER" : "6lowpanInterface_router.json",
     "THREAD_END_DEVICE" : "ThreadInterface_end_device.json",
     "THREAD_ROUTER" : "ThreadInterface_router.json",
-    "NO_NETWORK": "no_network.json"
+    "NO_NETWORK": "no_network.json",
+    "MAC_TESTER": "MACTester.json"
```

Comment:
I would say "MAC tester" may be clear enough in context of Nanostack, but is too generic for Mbed OS as a whole - the name should say something about Nanostack or 802.15.4